### PR TITLE
Add files for ROBB Smarrt ROB_200-081-0 device

### PR DESCRIPTION
**Pull request by azogtoth**
Changes made as per instructions

### CODE REVIEW

File: docs/devices/ROB_200-081-0.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+| **Zigbee Model**| ROB_200-081-0                         |
+
+## Installation
+1. Ensure Zigbee2MQTT is properly set up and running.
+2. Add the following definition to your Zigbee2MQTT `devices.js` file:
+
+```javascript
+const { deviceEndpoints, battery, commandsOnOff, commandsLevelCtrl, commandsColorCtrl } = require('zigbee-herdsman-converters/lib/modernExtend');
+
+const definition = {
```
Comment (Koenkk) [inline docs/devices/ROB_200-081-0.md:24]: Please make a pull request to add out-of-the-box support by clicking [here](https://github.com/Koenkk/zigbee-herdsman-converters/edit/master/src/devices/robb.ts)

Comment (azogtoth) [reply docs/devices/ROB_200-081-0.md:24]: Done (I think). How do I proceed now for this pull request. Have patience with me, as this is my first ever github contribution.

Comment (Koenkk) [reply docs/devices/ROB_200-081-0.md:24]: Looks good! I think `## Notes` up to notes can be removed (as it will part of the auto generated docs)

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**